### ISSUE TITLE
death.attack.no_water_for_gills

### ISSUE THREAD

**Issue by jadenquinn** [apace100/origins-fabric#603]
Basically, my friend died on a server I made for her, and this is what it said in console:
[20:18:12] [Server thread/INFO]: death.attack.no_water_for_gills
[20:18:24] [Server thread/INFO]: <theeternalsun404> fucccck

I don't know how she died, but I'm sure you can figure it out.

**Comment by eggohito** [comment]
`death.attack.no_water_for_gills` is the translation key for the death message for when a player that has the Merling origin dies if they haven't been in water for too long. It translates to *"\<username\> didn't manage to keep wet"*

**Comment by jadenquinn** [comment]
> `death.attack.no_water_for_gills` is the translation key for the death message for when a player that has the Merling origin dies if they haven't been in water for too long. It translates to *"\<username\> didn't manage to keep wet"*

Yup, although it doesn't say the translated bit in chat, it said exactly what it said in console.

**Comment by jadenquinn** [comment]
To reiterate, this is still a bug and it didn't just say that in console, it says it in-game.
